### PR TITLE
fix transitive orderbook ask/bid direction

### PR DESCRIPTION
**Pull request by nlordell**
Fixes the direction of ask/bid orders (again :rofl:).

:warning: Please review with caution! :warning: 

### Test Plan

Updated unit tests by flipping ask and bid orders without changing values. They seem to pass!

### CODE REVIEW

**Review by e00E — APPROVED** [review]
Tested with price estimator without and with this PR. Looks correct.

**Review by fleupold — APPROVED** [review]
I'm sorry I didn't notice this before 😢

**Comment by nlordell** [comment]
> I'm sorry I didn't notice this before cry

No worries! These logic bugs are super subtle and hard to find.

File: pricegraph/src/lib.rs [hunk 1]
```diff
@@ -32,7 +32,7 @@ impl TransitiveOrderbook {
     pub fn ask_prices(&self) -> impl DoubleEndedIterator<Item = (f64, f64)> + '_ {
         self.asks
             .iter()
-            .map(|order| ((order.buy / order.sell) * FEE_FACTOR, order.sell))
+            .map(|order| ((order.sell / order.buy) / FEE_FACTOR, order.buy))
```
Comment (fleupold) [inline pricegraph/src/lib.rs:35]: Trying to find the issue (cf #1037). Is this change correct? Asks sell quote and buy base. E.g. in ETH-USD they sell USD for ETH. The price (in quote) should be around $200, but if I sell 1ETH for $200 this evaluates to 0.005 (which is backwards).

Same for bids.

**Pull request merged**